src/invision/store/actions/add-movie.action.js

```diff
@@ -0,0 +1,22 @@
+import { ADD_MOVIE } from "./action.types";
```

Comment:
Usually, all actions for one reducer are collected to the one file. If you want to split action functions to the separate files, put them in one folder that may be easily connected with reducer

Reply:
Sure. I will follow this hereafter.